grpc/Dockerfile.client

```diff
@@ -0,0 +1,21 @@
+FROM golang:latest AS stage
```

Comment:
To avoid image pull throttling, we are serving all images that our samples require from our public ICR namespace. 
```suggestion
FROM icr.io/codeengine/golang:latest AS stage
```